homework/homework2/src/App.js

```diff
@@ -54,14 +55,16 @@ function App () {
         deadline: '',
         done: false,
       });
-    } else {window.alert('Please enter a todo');
+    } else {window.alert('Please enter a todo description and select a deadline');
```

Comment:
What I meant is to create multiple `window.alert`s, so that based on which field is missing you get a different error message.